---
title: "Analysing High Throughput Sequencing Data"
---

<head>

```{=html}
<script src="https://kit.fontawesome.com/ece750edd7.js" crossorigin="anonymous"></script>
```
</head>

```{r global_options, include=FALSE}
knitr::opts_chunk$set(warning=FALSE, message=FALSE)
```

------------------------------------------------------------------------

This is the homepage for the Analysing High Throughput Sequencing Data course run by the Bioinformatics Core at the [Discovery Research Platform for Hidden Cell Biology](https://www.ed.ac.uk/biology/drp).

<br>

![](images/DRP_banner_transparent2.png)

</br>

This workshop is designed as an introduction to ChIP-seq data analysis and will guide you through the following steps:

### Processing data

-   Assessing raw sequence data and performing quality control

-   Aligning sequence reads to a reference genome

-   Filtering alignments for further analysis

-   Visualising alignments on a genome browser

### Analysis

-   Assessing the quality of your ChIP-seq experiment

-   Summarising ChIP-seq profiles across genes

-   Peak calling

-   Motif discovery

-   Scripting

There is no such thing as a default pipeline. Although we mostly use standard parameters in this tutorial we hope to make you aware of the considerations you should take at each step. Make sure you understand your data and where it has come from. Use the correct tools for your dataset and read the tool documentation to see how different parameters affect your output!

------------------------------------------------------------------------

For more information contact [Shaun Webb](mailto:shaun.webb@ed.ac.uk).
